src/pages/home/report/ReportActionsView.js

```diff
@@ -70,7 +70,7 @@ function ReportActionsView(props) {
     const reactionListRef = useContext(ReactionListContext);
     const didLayout = useRef(false);
     const didSubscribeToReportTypingEvents = useRef(false);
-    const isFetchNewerWasCalled = useRef(false);
+    const isFirstRender = useRef(false);
```

Comment:
The initial value should be true here (and we need to inverse the logic in loadNewerChats)

Reply:
👍🏻